addons/event/__manifest__.py

```diff
@@ -36,7 +36,7 @@
         'views/res_config_settings_views.xml',
         'views/event_templates.xml',
         'views/res_partner_views.xml',
-        'views/event_tag_views.xml'
+        'views/event_tag_views.xml',
```

Comment:
Useless change, even if correct it's best to avoid adding steps when blaming stuff imo